### ISSUE TITLE
false error "message should be lowercased" when a message starts with a space.

### ISSUE THREAD

**Issue by slashformotion** [go-simpler/sloglint#104]
The error "message should be lowercased" is triggered when a message with a space is sent. Is this the expected behavior ?

```go
// http handler for the health service
//
// it is intended to be used on any http server
func (hh *HealthServer) handleFunc(w http.ResponseWriter, r *http.Request) {
	hh.log.DebugContext(r.Context(), " received request on /health")
```

the following error is returned: 
```
internal/health/health.go:151:35: message should be lowercased (sloglint)
	hh.log.DebugContext(r.Context(), " received request on /health",
	                                 ^
1 issues:
* sloglint: 1
```

I am using golangci-lint: `golangci-lint has version 2.6.1 built with go1.25.3 from e3b3bac5 on 2025-11-04T11:05:57Z`
with this configuration:

```yml
version: "2"
run:
  modules-download-mode: readonly
  issues-exit-code: 1
  tests: false
output:
  formats:
    text:
      path: stdout
linters:
  default: none
  enable:
    - sloglint
    - # other linter here
  settings:
    sloglint:
      # Enforce using attributes only (overrides no-mixed-args, incompatible with kv-only).
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#attributes-only
      # Default: false
      attr-only: true
      # Enforce not using global loggers.
      # Values:
      # - "": disabled
      # - "all": report all global loggers
      # - "default": report only the default slog logger
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#no-global
      # Default: ""
      no-global: "all"
      # Enforce using methods that accept a context.
      # Values:
      # - "": disabled
      # - "all": report all contextless calls
      # - "scope": report only if a context exists in the scope of the outermost function
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#context-only
      # Default: ""
      context: "scope"
      # Enforce using static values for log messages.
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#static-messages
      # Default: false
      static-msg: true
      # Enforce message style.
      # Values: lowercased, capitalized
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#message-style
      # Default: ""
      msg-style: lowercased
      # Enforce using constants instead of raw keys.
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#no-raw-keys
      # Default: false
      no-raw-keys: false # FIXME: set to true later
      # Enforce key naming convention.
      # Values: snake, kebab, camel, pascal
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#key-naming-convention
      # Default: ""
      key-naming-case: snake
      # Enforce not using specific keys.
      # https://github.com/go-simpler/sloglint?tab=readme-ov-file#forbidden-keys
      # Default: []
      forbidden-keys:
        - time
        - level
        - msg
        - source
```
